cfme/tests/webui/test_splitter.py

```diff
@@ -72,9 +73,10 @@ def test_pull_splitter_persistence(request, appliance, model_object, destination
     navigate_to(appliance.server, 'Dashboard')
     try:
         navigate_to(model_object, destination)
-    except (TypeError, CannotScrollException):
+    except (TypeError, CannotScrollException, TimedOutError):
```

Comment:
are you sure it won't hide some other TimedOutError ?

Reply:
Currently it works fine.
Actually, I got this error `TimedOutError('Could not do Waiting for view [ConfigurationView] to display at .../cfme/utils/appliance/implementations/ui.py:562 in time',)`. @izapolsk Is there any better way to pass this exception? it happens because `is_displayed` is based on accordions and in this test it's fine

Reply:
hm, okay. let's merge it as it then.